spec/models/company_spec.rb

```diff
@@ -96,13 +93,12 @@
 
   end
 
-  describe 'address_visible scope' do
-    it 'only returns companies that are complete' do
+  describe '.address_visible' do
```

Comment:
(Since a scope is a class method, you could have left the description the same.  Whatever your prefer is A-OK.)